.pa11yci.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+
+let options = {
+  defaults: {
+    // 'Section508', 'WCAG2A', 'WCAG2AA', or 'WCAG2AAA'
```

Comment:
* Section 508 - 4 tests fail, all are "H4 should be H3". Headings should follow logical nesting. Since it goes from H1 to H2 to H4.
* WCAG 2.1 A - All 32 files pass
* WCAG 2.1 AA - All 32 files pass
* WCAG 2.1 AAA - All 32 files fail with 3,942 errors

AAA is very strict and often only used on sites specifically aimed at an audience known to require additional accommodations. Generally when attempting to satisfy AAA you will need to get creative with the design and color choices to meet the very high contrast ratio requirements.

With that said, you could still turn this setting on locally, and see if there are any easy wins that could be achieved to get the 3,942 lowered.

Reply:
Yeah, I think government sites require AA which is a good target. AAA might be nice just to see what could be better, but maybe that's just a warning and not an error. What do you think?

Reply:
US Federal websites require Section 508 compliance, which is the weakest in the set, but pretty similar to A.

I don't think you can enforce AA and warn on AAA stuff, since most of it is just degrees of difference, like color contrast being 4.5:1 in AA or 7:1 in AAA.

AA is good enough for now, could be good to temporarily enable AAA locally to just fix low hanging fruit and then switch back to AA before committing. This can be done in the future.